frontend/tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
```

Comment:
`for ... of` loops require you to bump the target up from ES5 to ES2017 or else TS won't compile. This shouldn't affect any users since everyone's running ES2017 compatible browsers anyway.